plugins/modules/mysql_user.py

```diff
@@ -74,6 +74,19 @@
       - Whether binary logging should be enabled or disabled for the connection.
     type: bool
     default: yes
+  force_context:
+    description:
+      - Sets the С(mysql) system database as context for the executed statements (it will be used
+        as a database to connect to). Useful if you use binlog / replication filters in MySQL as
+        per default the statements can not be caught by a binlog / replication filter, they require
+        a database to be set to work.
```

Comment:
```suggestion
        a database to be set to work, otherwise the replication can break down.
```
Or something like that (not sure if we also need a period after `otherwise`)